### PR TITLE
oneDNN v3.13 release notes

### PR DESCRIPTION
**Pull request by vgvozdeva**
This PR contains draft release notes for oneDNN v3.13. Please review and update content relevant to your contributions as necessary.

Quick reminder about release notes structure:

- "Performance Optimizations" section lists user visible performance improvements for existing functionality in the library. No changes in the user code is required to take advantage of these.

- "Functionality" section lists new user visible functionality that was not available in the library at all or not implemented on a particular platform.

- "Usability" section lists user visible quality of life improvements. For example profiling and debug capabilities or improvements related to application compatibility.

- "Breaking changes" section highlights changes that may cause applications to break and require changes on the user end. For example bumping minimal supported version of a dependency.

- "Known issues" section highlights bugs that we want users to be informed about.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* Introduced [user-managed scratchpad] support in Graph API.
+
+[user-managed scratchpad]: https://uxlfoundation.github.io/oneDNN/v3.13/dev_guide_graph_scratchpad.html
+
+## Intel Graphics
+* Refactored verbose profiling on Intel GPUs to avoid spurious synchronizations with SYCL or OpenCL runtimes. New implementation reports device-time instead of host time and compatible with SYCL Graph record/replay mode. 
+* Reduced memory consumption of [Gated MLP subgraph] with Graph API.
+* Enabled interoperability with SYCL Graph native recording mode for Intel GPUs.
+* Introduced `ONEDNN_ZE_INCLUDE_DIR` and `ONEDNN_OCL_INCLUDE_DIR` build knobs to use Level Zero or OpenCL headers from a user defined location instead of vendored in version.
+* **[Experimental]** Introduced  support for [persistent cache] with Level Zero runtime on GPU. Level Zero support is experimental.
```
[the remark this answers is not in the document]
Comment (dzarukin) [reply RELEASE_NOTES.md]: Technically, Level Zero runtime support is not marked as experimental from API point of view...

[the remark this answers is not in the document]
Comment (vpirogov) [reply RELEASE_NOTES.md]: It's conveniently tucked behind a build knob though :) Maybe we should clarify this fact in the documentation.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Intel Graphics
+* Improved performance for future integrated GPUs based on Xe3p-LPG architecture (codename Nova Lake P).
+* Improved `u8`/`s8` convolution performance on Intel Arc B-series graphics.
+* Improved `f16` and `u8`/`s8` matmul performance with `u8`/`s8` and `u4`/`s4` weights in non-transposed layout.
+
+## AArch64 Processors
+TBD
+
+## RISC-V Processors
```
Comment (zhangfeiv0) [inline RELEASE_NOTES.md:27]: * Improved `f32` convolution, matmul, inner product, binary, eltwise, pooling, batch normalization, and group normalization primitive performance on processors with `V` extension support.
* Improved `f16` matmul, binary, eltwise, pooling, softmax, and layer normalization primitive performance on processors with `Zvfh` extension support.
* Improved `bf16` matmul primitive performance on processors with `Zvfbfwma` extension support.

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Extended SYCL Graph validation mode in benchdnn with support for native recording mode. This mode is enabled using `--execution-mode=native_graph` knob.
+* Enabled SYCL recording mode validation `--execution-mode=graph` for benchdnn `--graph` driver.
+* Introduced benchdnn knob `--mode=S` to improve performance validation speed in simulation or emulation environments.
+* Improved GPU performance reporting for mode=F by stabilizing measurement methodology and reducing inaccuracies caused by cache effects and run‑to‑run variability.
+
+# Deprecated Functionality
+* [BLAS-like API] including `dnnl::sgemm`, `dnnl::gemm_u8s8s32`, and `dnnl::gemm_s8s8s32` functions is deprecated
+  and will be removed in future releases. If you are using this API consider switching to [matmul primitive].
+* `f4_e3m0` data type is deprecated and will be removed in future releases.
+* Optimizations for Intel Iris Xe MAX Graphics and Intel Graphics included with 11th-14th Generation Intel Core Processors are deprecated and will be removed in the future releases.
```
[the remark this answers is not in the document]
Comment (vgvozdeva) [reply RELEASE_NOTES.md]: Added to latest commit

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+# Usability
+
+## Common
+* Introduced [user-managed scratchpad] support in Graph API.
+
+[user-managed scratchpad]: https://uxlfoundation.github.io/oneDNN/v3.13/dev_guide_graph_scratchpad.html
+
+## Intel Graphics
+* Refactored verbose profiling on Intel GPUs to avoid spurious synchronizations with SYCL or OpenCL runtimes. New implementation reports device-time instead of host time and compatible with SYCL Graph record/replay mode.
```
[the remark this answers is not in the document]
Comment (vgvozdeva) [reply RELEASE_NOTES.md]: Added to latest commit

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+# Validation
+* Extended SYCL Graph validation mode in benchdnn with support for native recording mode. This mode is enabled using `--execution-mode=native_graph` knob.
+* Enabled SYCL recording mode validation `--execution-mode=graph` for benchdnn `--graph` driver.
+* Introduced benchdnn knob `--mode=S` to improve performance validation speed in simulation or emulation environments.
+* Improved GPU performance reporting for mode=F by stabilizing measurement methodology and reducing inaccuracies caused by cache effects and run‑to‑run variability.
+
+# Deprecated Functionality
+* [BLAS-like API] including `dnnl::sgemm`, `dnnl::gemm_u8s8s32`, and `dnnl::gemm_s8s8s32` functions is deprecated
+  and will be removed in future releases. If you are using this API consider switching to [matmul primitive].
```
[the remark this answers is not in the document]
Comment (vgvozdeva) [reply RELEASE_NOTES.md]: Added to latest commit

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+## Common
+* Introduced [user-managed scratchpad] support in Graph API.
+
+[user-managed scratchpad]: https://uxlfoundation.github.io/oneDNN/v3.13/dev_guide_graph_scratchpad.html
+
+## Intel Graphics
+* Refactored verbose profiling on Intel GPUs to avoid spurious synchronizations with SYCL or OpenCL runtimes. New implementation reports device-time instead of host time and compatible with SYCL Graph record/replay mode. 
+* Reduced memory consumption of [Gated MLP subgraph] with Graph API.
+* Enabled interoperability with SYCL Graph native recording mode for Intel GPUs.
+* Introduced `ONEDNN_ZE_INCLUDE_DIR` and `ONEDNN_OCL_INCLUDE_DIR` build knobs to use Level Zero or OpenCL headers from a user defined location instead of vendored in version.
```
[the remark this answers is not in the document]
Comment (vgvozdeva) [reply RELEASE_NOTES.md]: Added to latest commit

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+* **[Experimental]** Introduced  support for [persistent cache] with Level Zero runtime on GPU. Level Zero support is experimental.
+
+[Gated MLP subgraph]: https://uxlfoundation.github.io/oneDNN/v3.13/dev_guide_graph_gated_mlp.html
+[persistent cache]: https://uxlfoundation.github.io/oneDNN/v3.13/dev_guide_persistent_cache.html
+
+# Validation
+* Extended SYCL Graph validation mode in benchdnn with support for native recording mode. This mode is enabled using `--execution-mode=native_graph` knob.
+* Enabled SYCL recording mode validation `--execution-mode=graph` for benchdnn `--graph` driver.
+* Introduced benchdnn knob `--mode=S` to improve performance validation speed in simulation or emulation environments.
+* Improved GPU performance reporting for mode=F by stabilizing measurement methodology and reducing inaccuracies caused by cache effects and run‑to‑run variability.
```
[the remark this answers is not in the document]
Comment (vgvozdeva) [reply RELEASE_NOTES.md]: Added to latest commit

File: RELEASE_NOTES.md (hunk 2 above)
Comment (vgvozdeva) [reply RELEASE_NOTES.md:27]: Thank you for input! Added in latest commit

**Review by vpirogov — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
+# Performance Optimizations
+## Intel 64/AMD64 Processors
+* Improved performance on future Intel Core Ultra processors with Intel AVX10.2 instruction set support (codename Nova Lake).
+* Improved performance of `f8` Scaled Dot Product Attention (SDPA) subgraph with Graph API.
```
Comment (TaoLv) [inline RELEASE_NOTES.md]:
```suggestion
* Improved performance of `f8` [quantized Scaled Dot Product Attention (SDPA)] subgraph with Graph API.
* 

[quantized Scaled Dot Product Attention (SDPA)]: https://uxlfoundation.github.io/oneDNN/v3.13/dev_guide_graph_sdpa_quantized.html
```

**Review by TaoLv — APPROVED** [review]

**Review by Sqvid — APPROVED** [review]

**Review by tprimak — APPROVED** [review]

**Review by rjoursler — APPROVED** [review]
